### PR TITLE
Update to Version 1.5.5

### PR DESCRIPTION
**Pull request by TayloredSoftware**

### CODE REVIEW

**Comment by LtFrankDrebin1975** [comment]
We really need those updates...

**Comment by MysticRyuujin** [comment]
Branch has merge conflicts, I'm happy to merge if they get fixed

**Comment by TayloredSoftware** [comment]
> Branch has merge conflicts, I'm happy to merge if they get fixed

Apologies submitted the corrections.

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ NOTE: Only the switches for MySQL Host, MySQL Port and Guacamole Database are av
 
 ### Download file directly from here:
 
-`wget https://raw.githubusercontent.com/MysticRyuujin/guac-install/master/guac-upgrade.sh`
+`wget https://raw.githubusercontent.com/TayloredSoftware/guac-install/master/guac-upgrade.sh`
```
Comment (quickchase) [inline README.md:128]: Why this change?

Comment (TayloredSoftware) [reply README.md]: Just caught that. I used on my own repository. Just submitted the default back.

**Review by MysticRyuujin — APPROVED** [review]
